src/custom/pages/App/index.tsx

```diff
@@ -91,7 +91,11 @@ function createRedirectExternal(url: string) {
 const Loading = <LoadingWrapper>Loading...</LoadingWrapper>
 
 const getShowBannerState = (key: string) => {
-  return JSON.parse(localStorage.getItem(key) || 'true') // show banner by default
+  const localStorageValue = localStorage.getItem(key)
+  if (localStorageValue && typeof localStorageValue === 'string') {
```

Comment:
mm no still a bit weird, consider this:
```js
const jsonValue = localStorage.getItem('some_key_for_item_that_doesnt_exist') // returns string value or null
const showBanner = Boolean(JSON.parse(jsonValue)) // true || false
```

Reply:
you can try it yourself in console:
```js
Boolean(JSON.parse(localStorage.getItem('some_bs_key')))
```

Reply:
for the sake of time my suggestion is to rerwrite this as 
```ts
const getShowBannerState = (key: string) => {
  const localStorageValue = localStorage.getItem(key)

  // item doesn't exist, show banner (true)
  if (localStorageValue === null) return true

  // else return localstorage state (!! for type safety)
  return !!JSON.parse(localStorageValue)
}
```

note: i just realised u are checking `null` as in, no state, as show banner aka `true`- so can be rewritten as above

Reply:
Thanks for checking into this. Yes, the intention was to return `true` if no state exists. Just committed your suggestion.